Client/Frontend/Browser/Tabs/RecentlyClosedTabs/RecentlyClosedTabsView.swift

```diff
@@ -0,0 +1,155 @@
+// Copyright 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import SwiftUI
+import Shared
+import BraveShared
+import Data
+import CoreData
+import BraveUI
+import BraveCore
+
+struct RecentlyClosedTabsView: View {
+  @Environment(\.presentationMode) @Binding private var presentationMode
+
+  @State var recentlyClosedTabs: [RecentlyClosed] = []
+  @State private var recentlyClosedLoading = true
+  
+  @State private var showClearDataPrompt: Bool = false
+  var onDismiss: ((Bool) -> Void)?
+  var onRecentlyClosedSelected: ((RecentlyClosed) -> Void)?
+  
+  private let tabManager: TabManager?
```

Comment:
Because I wanted be able to define it without tabmanager. It is used for clearing outdated recently closed objects. But if you have some strong opinion about this I can make the change.